src/pages/auth/PrivateViews.js

```diff
@@ -1,12 +1,60 @@
-import React, { Fragment } from 'react';
+import React, { Component, Fragment } from 'react';
+import { object } from 'prop-types';
+import { Trans } from '@lingui/macro';
 import NavBar from 'pages/dashboard/components/navBar/NavBar';
-import { navConfiguration } from 'config/navigation';
-
-export default function PrivateViews({ children, ...rest }) {
-  return (
-    <Fragment>
-      <NavBar configuration={navConfiguration} currentPath={rest.location.pathname} />
-      <main className="min-h-screen bg-secondary-lightest">{children}</main>
-    </Fragment>
-  );
+import Button from 'components/button/Button';
+import { Panel, PanelContent } from 'components/panel/Panel';
+
+export default class PrivateViews extends Component {
+  static propTypes = {
+    auth: object,
```

Comment:
Maybe we could add `object.isRequired`?

Reply:
That's a good idea

Reply:
**Reminder to add isRequired**

Reply:
that's very weird... I do have a commit adding the required: https://github.com/Coding-Coach/coding-coach/pull/312/commits/5f3a026557e92294528b69069abd63573e6f7bf8 🤔